lib/cloud/watchers/watcher.go

```diff
@@ -98,6 +119,11 @@ func NewWatcher(ctx context.Context, config WatcherConfig) (*Watcher, error) {
 // subscribe to events such as created/removed instances and tag changes, but
 // this will require Teleport to have more AWS permissions.
 func (w *Watcher) Start() {
+	go w.startEC2()
+	w.start()
+}
```

Comment:
should we keep the database vs EC2 watcher separate? I assume they will be used for different services.